catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/main/java/org/codice/ddf/spatial/ogc/wfs/v110/catalog/source/WfsFilterDelegate.java

```diff
@@ -1097,14 +1109,11 @@ private FilterType buildSingleGeospatialFilter(
 
     Coordinate[] coordinates = getCoordinatesFromWkt(wkt);
     if (coordinates != null && coordinates.length > 0) {
-      StringBuilder coordString = new StringBuilder();
-
-      for (Coordinate coordinate : coordinates) {
-        coordString.append(coordinate.y).append(",").append(coordinate.x).append(" ");
-      }
+      String coordinateString =
```

Comment:
✏️ Similar to my above comment, create a class that encapsulates the data and methods.